html/table.html

```diff
@@ -145,24 +155,124 @@
         const clicked_color = "blue";
         const ID_FIELD = 0;
         // Refactoring selected nodes to make it easier to update the legend and selected nodes on dom changes
-        //const SELECTED_NODES = new Set();
         const right_mouse = 3;
         const center_mouse = 2;
         const left_mouse = 1;
         const CIRCLE_SIZE_SELECTED = 6;
         const INNER_NODE_SIZE = 4;
         const LEAF_NODE_SIZE = 6;
-        const DECIMAL_PLACES = 4;
+        var DECIMAL_PLACES = 1;
```

Comment:
Kudos for making 1 decimal place as a default value as per PulseNet Canada recommendations. We also need to maybe default to 0 decimal places if in Newick file all distance values are integers instead of floats. Also max number of decimal points should be limited by the newick file distance values max resolution. This could be implemented as more smarter approach later on